WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/adapters/viewholders/ScanStateViewHolder.kt

```diff
@@ -0,0 +1,23 @@
+package org.wordpress.android.ui.jetpack.scan.adapters.viewholders
+
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.scan_list_scan_state_item.*
+import org.wordpress.android.R
+import org.wordpress.android.ui.jetpack.scan.ScanListItemState
+import org.wordpress.android.ui.jetpack.scan.ScanListItemState.ScanState
+import org.wordpress.android.ui.utils.UiHelpers
+
+class ScanStateViewHolder(
+    private val uiHelpers: UiHelpers,
+    parent: ViewGroup
+) : ScanViewHolder(R.layout.scan_list_scan_state_item, parent) {
+    override fun onBind(itemUiState: ScanListItemState) {
+        val scanState = itemUiState as ScanState
+
+        scan_state_icon.setImageResource(scanState.scanIcon)
```

Comment:
💡 Idea: We might want to consider using an `ImageManager.load(imageView: ImageView, @DrawableRes resourceId: Int, scaleType: ScaleType = CENTER)` here. I think it'd be great if we used it for all image loading. Wdyt?

Reply:
Didn't know that this existed, thanks for sharing Jirka! Btw, since this is Kotlin, why not using this extension method pattern for loading images. Has this been discussed or implemented here yet?

Reply:
Noticed `imageView.setImageResource(activity.icon.drawable)` in [EventItemViewHolder](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/EventItemViewHolder.kt#L55
) and it seemed like icon drawables were directly set using `setImageResource`. But I liked your idea and changed to it in 6522dff.